server/package.json

```diff
@@ -35,6 +35,7 @@
     "koa": "^2.4.1",
     "koa-router": "^7.1.0",
     "koa-static": "^3.0.0",
+    "lodash.groupby": "^4.6.0",
```

Comment:
Do we have an indication of how much bundle size this adds?

Reply:
Stat size: 63KB
Parsed size: 12.44KB
Gzipped size: 4.55KB